### DISCUSSION TITLE
We have 1 hour remaining. Rest in peace lazer iOS.

### DISCUSSION THREAD

**Discussion by Jacob1863** [ppy/osu#29976] [Q&A]
I just got back from school, and I checked the testflight app, and an hour remaining is left.
![IMG_3401](https://github.com/user-attachments/assets/88c2242f-37e4-4def-8866-3ded20c1259e)

**Comment by Jacob1863** [comment]
I forgot sideloading is a thing. Welp, AltStore I guess…

**Comment by peppy** [comment]
Stay tuned, we likely have another build coming soon.

**Reply by Jacob1863** [reply]
I can wait, since I sideloaded the version “2024.625.2”. Thanks for reminding me!
